### PR TITLE
Dynamic Content SEO

### PR DESCRIPTION
**Pull request by rkdavidson**
# About
Lets the SEO values be populated dynamically based off content items.
Enhances the `SEO` component to accept a more robust set of `meta` tags, and modifies `<ContentSingle>` to use it (via `<ContentLayout>`).

# Testing
Go to any content item (via **Discover**, etc) and inspect the pages' `<meta>` tags.
Verify they match the content as expected. For bonus, share via Twitter and Facebook or use a tool like [opengraph](https://www.opengraph.xyz/).

Items with `videos` should show an `og:video` tag. Same with `author`.
Default fallback values should be used for things like `keywords`, `description`, etc.

# Screenshots

video
generic
article

### CODE REVIEW

**Review by drewbarontini — COMMENTED** [review]
Looks good! I took it out of the `Draft` state since you requested reviews. I'll approve once the conflicting file is fixed 👍

**Comment by rkdavidson** [comment]
@drewbarontini Ah thanks, should have held off on requesting reviews then — was more like "this is ready for pre-review while buttoning details up", so I'll hang tight on requesting until PR is ready for _full_ review in future.

Will have this updated in a few.

**Comment by rkdavidson** [comment]
@drewbarontini / @jordanwade Ready for review ✅

File: components/SEO/SEO.js [hunk 1]
```diff
@@ -2,31 +2,54 @@ import React from 'react';
...
 function getPageTitle(title) {
   if (title === DEFAULT_TITLE || title === 'Home') return DEFAULT_TITLE;
   return `${title} - ${DEFAULT_TITLE}`;
 }
 
 function SEO(props = {}) {
-  const title = getPageTitle(props.title);
-
+  const pageTitle = getPageTitle(props.title);
+  console.log('props.meta:', props.meta);
```
Comment (drewbarontini) [inline components/SEO/SEO.js:19]: 🔥 ?

**Review by drewbarontini — COMMENTED** [review]
One `console.log`, but this is looking good 👍

**Comment by asleepingpanda** [comment]
Updated the branch from `main`. Otherwise, this looks great to me!

**Review by asleepingpanda — APPROVED** [review]

**Pull request merged**